server/command_hooks.go

```diff
@@ -722,3 +762,89 @@ func (p *Plugin) executeAutocompleteTest(args *model.CommandArgs) *model.Command
 		Text:         fmt.Sprintf("Executed command: " + args.Command),
 	}
 }
+
+func (p *Plugin) executeCommandPreferences(args *model.CommandArgs) *model.CommandResponse {
+	fields := strings.Fields(args.Command)
+	command := ""
+	if len(fields) == 2 {
```

Comment:
Should that be 
```suggestion
	if len(fields) >= 2 {
```
?